dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -200,7 +205,7 @@ export default defineComponent({
   },
   mounted() {
     const dataId = this.route.query.templateDataId;
-    if (dataId !== undefined && typeof dataId === "string" && dataId !== "") {
+    if (dataId && typeof dataId === "string") {
```

Comment:
Is if (dataId) the same as if (dataId !==undefined && dataId !=="")  ?

Reply:
Yes, but technically it could be `dataId !== undefined && typeof dataId !== "string"` , this would crash, so there is an explicit condition on it being of type string. Usually `if (dataId)` should be enough.

Reply:
ok